src/commands/actors/index.ts

```diff
@@ -0,0 +1,9 @@
+import { ApifyCommand } from '../../lib/apify_command.js';
+
+export class ActorIndexCommand extends ApifyCommand<typeof ActorIndexCommand> {
+	static override description = 'Commands are designed to be used with Actors.';
```

Comment:
Since this is an override, I guess it'll get glued to something else before printing to the console? On its own, the wording would be kinda weird.

Reply:
Yep, its printed when you run `apify actors` alone

Reply:
So what's the exact thing that gets printed in that case?

Reply:
![Code - 2024-09-06 at 13 45 59@2x](https://github.com/user-attachments/assets/88343e2b-15de-4652-acee-a0f16ece5f24)

When you have commands those get listed too

![Code - 2024-09-06 at 13 46 29@2x](https://github.com/user-attachments/assets/846ab4fb-3962-4c5c-9432-479a4fc924c8)

Reply:
Gotcha, thanks. Then I'd probably reword those descriptions along the lines of "Commands for working with Actors" or "for management of Actors"

Reply:
Thats more of a @netmilk task, I just did what we did for other namespaces. Agreed it could use better copies but I wouldn't wanna block feature implementation just bc of that

Reply:
I get it, but if we don't do it now, there will always be something more pressing than changing wording.

Reply:
Sure, but that can *hopefully* be done in parallel by Adam 👀. This also ties into @barjin's comments on #620 about wording tbh, we definitely need a pass on all wording and improve it

Reply:
Made https://github.com/apify/apify-cli/issues/637 to track this